cli/src/description_util.rs

```diff
@@ -251,13 +252,22 @@ pub fn description_template(
     Ok(output.into_string_lossy())
 }
 
+pub fn parse_user_name_email(author: &str) -> Result<(String, String), CommandError> {
```

Comment:
I hadn't thought about that earlier, but it could be moved to Clap `https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_parser`

Reply:
Done